addons/stock_account/views/stock_valuation_layer_views.xml

```diff
@@ -81,12 +105,23 @@
         </field>
     </record>
 
+    <record id="stock_valuation_layer_graph" model="ir.ui.view">
+        <field name="name">inventory.aging.graph</field>
+        <field name="model">stock.valuation.layer</field>
+        <field name="arch" type="xml">
+            <graph>
+                <field name="remaining_qty" type="measure"/>
+            </graph>
+        </field>
+    </record>
+
     <record id="stock_valuation_layer_action" model="ir.actions.act_window">
         <field name="name">Stock Valuation</field>
         <field name="res_model">stock.valuation.layer</field>
         <field name="view_mode">tree,form,pivot</field>
         <field name="view_id" ref="stock_valuation_layer_tree"/>
         <field name="domain">[('product_id.type', '=', 'product')]</field>
+        <field name="context">{'search_default_has_remaining_qty': True, 'search_default_group_by_product_id': True}</field>
```

Comment:
remove `has_remaining_qty` filter in `action_view_stock_valuation_layers`. It's not necessary and makes it show nothing sometimes